internal/stack/certs.go

```diff
@@ -26,6 +26,11 @@ var tlsServices = []string{
 	"logstash",
 }
 
+var tlsServicesServerless = []string{
+	"logstash",
+	"elastic-agent",
+}
```

Comment:
Previously, it was not needed a certificate for `elastic-agent`. Would it be ok to just generate the cert for `logstash` service? Or, is it required both ?